loops_lab.js

```diff
@@ -0,0 +1,57 @@
+
+
+
+//-------------------------------------
+let num = 5;
+while (num < 25) {
+  console.log(num);
+  num += 1;
+}
+
+for (let number = 0; number <= 10; number += 1) {
+  if (number % 2 !== 0) {
+    console.log(number + ' num is odd');
+  } else {
+    console.log(number + ' num is even');
+  }
+}
+
+let bynine = 99;
+while (bynine > 0) {
+  if (bynine % 9 !== 0) {
+    console.log(bynine  + ' is not divisible by 9');
+  } else {
+    console.log(bynine + 'is divisible by 9');
+  } bynine -= 1;
+}
+
+for (let i = 5; i <= 100; i += 10) {
```

Comment:
Interesting way of solving this problem but can you think of a solution where you can use modulo operator to solve the same problem?